sandpack-react/src/types.ts

```diff
@@ -54,7 +57,12 @@ export interface SandpackState {
   loadingScreenRegisteredRef: React.MutableRefObject<boolean>;
 }
 
-export type SandpackStatus = "initial" | "idle" | "running" | "timeout";
+export type SandpackStatus =
```

Comment:
Have you faced any issue with this type?

Reply:
Yup, `done` is actually fired at the end after success in bundler. But wasn't present in the interface, so added it 👍